packages/aurora-erc20/src/bridged-erc20/sendToEthereum/index.ts

```diff
@@ -598,35 +606,26 @@ export async function checkBurn (
   }
 
   let nearBurnReceipt
-  const nep141Factory = options.nep141Factory ?? getBridgeParams().nep141Factory
-  try {
-    if (transfer.symbol !== 'ETH') {
-      nearBurnReceipt = await parseNep141BurnReceipt(nearBurnTx, nep141Factory, nearProvider)
-    } else {
-      // Withdraw ERC-20 ETH from a silo which doesn't use ETH as native currency.
-      nearBurnReceipt = await parseETHBurnReceipt(nearBurnTx, transfer.auroraEvmAccount ?? 'aurora', nearProvider)
-    }
-  } catch (e) {
-    if (e instanceof TransferError) {
-      return {
-        ...transfer,
-        errors: [...transfer.errors, e.message],
-        status: status.FAILED
-      }
-    }
```

Comment:
Why was this error handling removed?

Reply:
Cleaning up an error that doesn't exist anymore, if it fails checkStatus loop will try again.